tools/snippets/address_cache_all_all_seed.json

```diff
@@ -0,0 +1,3002 @@
+{
```

Comment:
this is really not a good place for this data.

also fwiw you could have gotten this data from https://iancoleman.io/bip39/

Reply:
Deleted in 2816b51. True that if somebody wants this, they can generate it